src/main.rs

```diff
@@ -37,7 +35,10 @@ use opentelemetry_otlp::ExportConfig;
 use runtime::PshEngineBuilder;
 use utils::check_root_privilege;
 
-fn main() -> anyhow::Result<()> {
```

Comment:
Turning `anyhow::Result<()>` to `Result<()>` just changes the code style.
A separate commit with message "import `anyhow::Result`" is appropriate to it.